src/lando/utils/tests/test_github.py

```diff
@@ -0,0 +1,227 @@
+        Date: Wed, 8 Oct 2025 17:51:16 +1100
+        Subject: [PATCH 3/5] add naughty try task config
+
+        ---
+         try_task_config.json | 0
+         1 file changed, 0 insertions(+), 0 deletions(-)
+         create mode 100644 try_task_config.json
+
+        diff --git a/try_task_config.json b/try_task_config.json
+        new file mode 100644
+        index 0000000..e69de29
+
+        From 1d9881143c8288d6d230869c8d5e2b26d12862cc Mon Sep 17 00:00:00 2001
+        From: User2 <user2@example.com>
+        Date: Wed, 8 Oct 2025 18:22:38 +1100
+        Subject: [PATCH 4/5] add non-empty b
+
+        ---
+         b | 1 +
+         1 file changed, 1 insertion(+)
+         create mode 100644 b
+
+        diff --git a/b b/b
+        new file mode 100644
+        index 0000000..e0b3f1b
+        --- /dev/null
+        +++ b/b
+        @@ -0,0 +1 @@
+        +bb
+
+        From d1adda9a692e3f362435b4f80ea19aa41c555e69 Mon Sep 17 00:00:00 2001
+        From: User2 <user2@example.com>
+        Date: Wed, 8 Oct 2025 18:26:29 +1100
+        Subject: [PATCH 5/5] add two more files
+
+        ---
+         1 | 1 +
+         2 | 1 +
+         2 files changed, 2 insertions(+)
+         create mode 100644 1
+         create mode 100644 2
+
+        diff --git a/1 b/1
+        new file mode 100644
+        index 0000000..d00491f
+        --- /dev/null
+        +++ b/1
+        @@ -0,0 +1 @@
+        +1
+        diff --git a/2 b/2
+        new file mode 100644
+        index 0000000..0cfbf08
+        --- /dev/null
+        +++ b/2
+        @@ -0,0 +1 @@
+        +2
+        """
+    ).lstrip()
+
+
+@pytest.fixture
+def github_pr_diff() -> str:
+    # curl -LO https://github.com/mozilla-conduit/test-repo/pull/1.diff
+    return dedent(
+        """
+        diff --git a/1 b/1
+        new file mode 100644
+        index 0000000..d00491f
+        --- /dev/null
+        +++ b/1
+        @@ -0,0 +1 @@
+        +1
+        diff --git a/2 b/2
+        new file mode 100644
+        index 0000000..0cfbf08
+        --- /dev/null
+        +++ b/2
+        @@ -0,0 +1 @@
+        +2
+        diff --git a/b b/b
+        new file mode 100644
+        index 0000000..e0b3f1b
+        --- /dev/null
+        +++ b/b
+        @@ -0,0 +1 @@
+        +bb
+        diff --git a/favico.ico b/favico.ico
+        new file mode 100644
+        index 0000000..7ae8144
+        Binary files /dev/null and b/favico.ico differ
+        diff --git a/test b/test
+        index 89b24ec..7bba8c8 100644
+        --- a/test
+        +++ b/test
+        @@ -1 +1,2 @@
+         line 1
+        +line 2
+        diff --git a/try_task_config.json b/try_task_config.json
+        new file mode 100644
+        index 0000000..e69de29
+        """
+    ).lstrip()
+
+
+def test_api_client_build_pr(
+    github_pr_response: str, github_pr_diff: str, github_pr_patch: str
+):
+    repo = mock.Mock
+    repo._github_repo_org = "mozilla-conduit"
+    repo.git_repo_name = "test-repo "
```

Comment:
Is the trailing space intentional?
```suggestion
    repo.git_repo_name = "test-repo"
```